Kanban.Desktop/Data.Sources.Common/Repository/IRepository.cs

```diff
@@ -5,7 +5,7 @@
 
 namespace Data.Sources.Common.Redmine
 {
-    public interface IRedmineRepository
+    public interface IRepository
```

Comment:
We lose not only readability here, we lose all significant information about wtf is this. I suggest ITicketRepository or leave it alone